Extensions/Xtensive.Orm.Localization.Tests/ProjectionToCustomTypesTests.cs

```diff
@@ -0,0 +1,55 @@
+using System.Linq;
+using System.Threading;
+using NUnit.Framework;
+using Xtensive.Orm.Localization.Tests.Model;
+
+namespace Xtensive.Orm.Localization.Tests
+{
+  [TestFixture]
+  public class ProjectionToCustomTypesTests : AutoBuildTest
+  {
+    protected override void PopulateDatabase()
+    {
+      using (var session = Domain.OpenSession()) {
+        using (var ts = session.OpenTransaction()) {
```

Comment:
Please do not use extra indentation in case of several ```using``` chain. Example:
```
using (var session = Domain.OpenSession())
using (var ts = session.OpenTransaction()) {

}
```

Reply:
done